snapcraft.yaml

```diff
@@ -335,17 +378,57 @@ parts:
       - rust
       - wasi-sdk
     build-packages:
+      - on amd64:
+        - to amd64:
+          - libasound2-dev
+          - libdbus-glib-1-dev
+          - libgtk-3-dev
+          - libgtk2.0-dev
+          - libpango1.0-dev
+          - libpulse-dev
```

Comment:
Pango, Pulse, Xkbcommon were not present in `build-packages:`, why were they added here?

Reply:
It is not related to cross-compilation, is it? If not, should be in a separate commit.

Reply:
It did show up during work on cross-compilation